app/models/employee.rb

```diff
@@ -7,6 +7,17 @@ class Employee < ApplicationRecord
   validates :private_number, numericality: { only_integer: true }
   scope :with_private_number, ->(number) { where(private_number: number) }
 
+  def absences
+    attendances = self.attendances.count
+    month_dates = self.current_month_dates
+    absences = month_dates - attendances
+    absences
+  end
+
+  def current_month_dates
+    (DateTime.now.beginning_of_month.beginning_of_day..DateTime.now.end_of_day).to_a.count
```

Comment:
What about using `days_in_month` ?
https://apidock.com/rails/v5.2.3/Time/days_in_month/class

Reply:
We changed this method, we don't knew can be refactor it to much because only we needed the current days in the month, we solved using a other more simple sentence, can you review it in this new amend.

thanks, DONE